client/container_copy.go

```diff
@@ -30,6 +30,7 @@ func (cli *Client) ContainerStatPath(ctx context.Context, containerID, path stri
 }
 
 // CopyToContainer copies content into the container filesystem.
+// Note that `content` must be a Reader for a TAR
```

Comment:
I'm struggling a bit with this revised sentence.  Should it be:

Note that `content` must be in TAR readable archive file format.

?